engine/components/physics/box2d/IgeEntityBox2d.js

```diff
@@ -1,8 +1,8 @@
 /**
  * Creates a new entity with box2d integration.
  */
-var IgeEntityBox2d = IgeEntity.extend({
-	classId: 'IgeEntityBox2d',
+var IgeEntityPhysics = IgeEntity.extend({
```

Comment:
file should also be renamed to `IgeEntityPhysics`

Reply:
I addressed this in the most recent commit.